clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -117,6 +117,9 @@ Improvements to clang-tidy
 
 - Improved :program:`clang-tidy` by accepting parameters file in command line.
 
+- Improved :program:`clang-tidy` by fixing incorrect configuration file path
```

Comment:
Note: to be honest all those 'Improved' release notes for same program could be merged together

Reply:
#122594